{
  "resourceType" : "ValueSet",
  "id" : "2.16.840.1.113762.1.4.1200.152",
  "meta" : {
    "versionId" : "11",
    "lastUpdated" : "2021-07-21T01:02:04.000-04:00",
    "profile" : [
      "http://hl7.org/fhir/StructureDefinition/shareablevalueset",
      "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-valueset-cqfm"
    ]
  },
  "extension" : [
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/valueset-author",
      "valueString" : "Cliniwiz Author"
    },
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/resource-lastReviewDate",
      "valueDate" : "2022-12-15"
    },
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/valueset-effectiveDate",
      "valueDate" : "2021-07-21"
    }
  ],
  "url" : "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1200.152",
  "identifier" : [
    {
      "system" : "urn:ietf:rfc:3986",
      "value" : "urn:oid:2.16.840.1.113762.1.4.1200.152"
    }
  ],
  "version" : "20210721",
  "name" : "COVID19Diagnosis",
  "title" : "COVID 19 Diagnosis",
  "status" : "active",
  "experimental" : false,
  "date" : "2021-07-21T01:02:04-04:00",
  "publisher" : "Cliniwiz Steward",
  "jurisdiction" : [
    {
      "extension" : [
        {
          "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
          "valueString" : "UNKNOWN"
        }
      ]
    }
  ],
  "purpose" : "(Clinical Focus: ),(Data Element Scope: ),(Inclusion Criteria: ),(Exclusion Criteria: )",
  "compose" : {
    "include" : [
      {
        "system" : "http://snomed.info/sct",
        "concept" : [
          {
            "code" : "840539006",
            "display" : "Disease caused by Severe acute respiratory syndrome coronavirus 2 (disorder)"
          }
        ]
      }
    ]
  },
  "expansion" : {
    "identifier" : "urn:uuid:463819b3-21bd-4a01-86f5-dda44b80316e",
    "timestamp" : "2023-06-29T16:37:25-04:00",
    "total" : 1,
    "offset" : 0,
    "parameter" : [
      {
        "name" : "count",
        "valueInteger" : 1000
      },
      {
        "name" : "offset",
        "valueInteger" : 0
      }
    ],
    "contains" : [
      {
        "system" : "http://snomed.info/sct",
        "version" : "http://snomed.info/sct/731000124108/version/20230301",
        "code" : "840539006",
        "display" : "Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"
      }
    ]
  }
}
